Dockerfile-debian.template

```diff
@@ -124,19 +123,23 @@ RUN set -ex; \
 # gpg key from https://nextcloud.com/nextcloud.asc
     gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys 28806A878AE423A28372792ED75899B9A724937A; \
     gpg --batch --verify nextcloud.tar.bz2.asc nextcloud.tar.bz2; \
-    tar -xjf nextcloud.tar.bz2 -C /usr/src/; \
+    tar -xjf nextcloud.tar.bz2 -C /var/www/; \
+    rm -rf /var/www/html; \
+    mv /var/www/nextcloud /var/www/html; \
```

Comment:
Why not tar them into `/var/www/html` directly?

Reply:
I think I just didn't manage to get it right :) If there is an agreement to merge this, I'll fix.